Junk4Dollars-app/Endpoints.swift

```diff
@@ -0,0 +1,8 @@
+public class ApiEndpoints {
```

Comment:
Idea for future refactor: Make `ApiEndpoints` an enum. You can have it inherit from `String` so that the cases have a `String` `rawValue`